.github/workflows/release.yml

```diff
@@ -35,40 +41,59 @@ jobs:
-          secrets: GITHUB_TOKEN=${{ secrets.GH_TOKEN }}
+      # - name: Build in Docker
+      #   uses: docker/build-push-action@v2
+      #   with:
+      #     context: ./
+      #     file: ./Dockerfile
+      #     builder: ${{ steps.buildx.outputs.name }}
+      #     push: false
+      #     load: true
+      #     tags: sealos:latest
+      #     build-args: |
+      #       ACTION=release
+      #       GITHUB_TOKEN=${{ secrets.GH_TOKEN }}
+      #     secrets: GITHUB_TOKEN=${{ secrets.GH_TOKEN }}
       # https://github.com/docker/build-push-action
 
-      - name: Copy Out Binaries
+      - name: Setup MultiArch Apt
         run: |
-          docker run -itd --name sealos --rm sealos:latest bash
-          docker cp sealos:/work/dist  .
+          sudo \cp .github/sources.list /etc/apt/sources.list
+          sudo dpkg --add-architecture arm
+          sudo apt update || true
 
-      - name: Build binaries to oss
-        env:
-          BUCKETNAME: ${{ secrets.BUCKETNAME }}
+      - name: Install Dependencies
         run: |
-          echo ${{ secrets.OSS_CONFIG }} |base64 --decode >> .ossutilconfig
-          wget http://gosspublic.alicdn.com/ossutil/1.6.19/ossutil64  && chmod 755 ossutil64
-          ./ossutil64 -c .ossutilconfig cp -f dist/sealos-amd64_linux_amd64/sealos oss://${BUCKETNAME:-sealyun-home}/sealos-4.0/${{ steps.prepare.outputs.tag_name }}/sealos-amd64
-          ./ossutil64 -c .ossutilconfig cp -f dist/sealos-arm64_linux_arm64/sealos oss://${BUCKETNAME:-sealyun-home}/sealos-4.0/${{ steps.prepare.outputs.tag_name }}/sealos-arm64
-          ./ossutil64 -c .ossutilconfig cp -f dist/sealos-amd64_linux_amd64/sealos oss://${BUCKETNAME:-sealyun-home}/sealos-4.0/latest/sealos-amd64
-          ./ossutil64 -c .ossutilconfig cp -f dist/sealos-arm64_linux_arm64/sealos oss://${BUCKETNAME:-sealyun-home}/sealos-4.0/latest/sealos-arm64
-          ./ossutil64 -c .ossutilconfig cp -f dist/sealctl_linux_amd64/sealctl oss://${BUCKETNAME:-sealyun-home}/sealos-4.0/${{ steps.prepare.outputs.tag_name }}/sealctl-amd64
-          ./ossutil64 -c .ossutilconfig cp -f dist/sealctl_linux_arm64/sealctl oss://${BUCKETNAME:-sealyun-home}/sealos-4.0/${{ steps.prepare.outputs.tag_name }}/sealctl-arm64
-          ./ossutil64 -c .ossutilconfig cp -f dist/sealctl_linux_amd64/sealctl oss://${BUCKETNAME:-sealyun-home}/sealos-4.0/latest/sealctl-amd64
-          ./ossutil64 -c .ossutilconfig cp -f dist/sealctl_linux_arm64/sealctl oss://${BUCKETNAME:-sealyun-home}/sealos-4.0/latest/sealctl-arm64
+          sudo apt install -y gcc-aarch64-linux-gnu upx \
+            libbtrfs-dev libgpgme-dev libdevmapper-dev \
+            libbtrfs-dev:arm64 libgpgme-dev:arm64 libdevmapper-dev:arm64
+
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v3
+        with:
+          args: release --rm-dist --release-notes=scripts/release/Note.md
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+
```

Comment:
add makefile make oss ?

Reply:
I'd like to test it before add steps to push binaries to oss.